src/language-html/embed.js

```diff
@@ -135,6 +136,17 @@ function embed(path, options) {
 
     case "front-matter":
       return (textToDoc) => printFrontMatter(node, textToDoc);
+
+    // TODO: This should work on `blockParameter`, but currently we don't have a parser for it.
+    case "block":
+      if (
+        !["if", "else if", "for", "switch", "case"].includes(node.name) ||
```

Comment:
ditto (Or we should introduce `const angularBlockNamesCanHaveParams = new Set(["if", "else if", "for", "switch", "case"])` ?)

Reply:
We'll need rewrite this part soon, let's keep it this way for now.